### DISCUSSION TITLE
Locked Facebook account

### DISCUSSION THREAD

**Discussion by danielraybor** [community/community#72300] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I’m doing my best, but at the beginning of learning all this. My fb account is locked and the people I’ve spoke with at Facebook haven’t figured it out yet. I believe I got a certain bug, possibly through a gmail account. And I believe it has something to do with lack of a token. Not positive. Would anyone be willing to speak with me about this? I have two business pages I really need back. Not to mention the last pics of my mother are on that profile. TIA

**Comment by ghostinhershell** [comment]
Hi @danielraybor,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you will need to reach out to Facebook support directly. The GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space is not the right place for this particular topic.

Best of luck!

**Comment by danielraybor** [comment]
Got you. I’m interested in learning it all so I would like to stay with GitHub . Thanks for responding

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by danielraybor** [comment]
I got all mine back

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by dinohit7** [comment]
Sincerely I do not know much about other hackers since I didn’t work with them but I can strongly recommend Marieconsultancyoz@gmail.com for all hacking and recovery  services because she has proven to be a great ethical hacker over the years, I read a lot of recommendation about her and this prompt me to contact her and she provide me a confidential result within 3hrs Text Marieconsultancyoz@gmail.com and instagram @Marie_consultancy

**Comment by ebndev** [comment]
Hi all / future viewers! I will be locking this discussion as all further comments have been spam.
